### PR TITLE
Add support to paste a lower dimensional image into a higher.

### PR DESCRIPTION
**Pull request by blowekamp**
Replace ITK's PasteImageFilter with the remote module SimpleFilters' NPasteImageFilter.

### CODE REVIEW

File: Code/BasicFilters/include/sitkPasteImageFilter.h [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+       *
+       * The axes with true values are set to 1, to fill the difference between the dimension of the input and source image.
+       * The number of true value in DestinationSkipAxes plus the DestinationImageDimension must equal the
+       * InputImageDimension.
+       *
+       * By default this array contains SourceImageDimension false values followed by true values for the remainder.
+       */
+      SITK_RETURN_SELF_TYPE_HEADER SetDestinationSkipAxes ( std::vector<bool> DestinationSkipAxes ) { this->m_DestinationSkipAxes = std::move(DestinationSkipAxes); return *this; }
+
+      /** Set/Get the an array describing which axes in the destination image to skip
```
Comment (dave3d) [inline Code/BasicFilters/include/sitkPasteImageFilter.h:107]: remove 'an'

File: Code/BasicFilters/include/sitkPasteImageFilter.h [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+       *
+       * The axes with true values are set to 1, to fill the difference between the dimension of the input and source image.
+       * The number of true value in DestinationSkipAxes plus the DestinationImageDimension must equal the
```
Comment (dave3d) [inline Code/BasicFilters/include/sitkPasteImageFilter.h:110]: values

**Review by dave3d — COMMENTED** [review]
As far as I can tell, when I ran the tests, the Python test never gets called.

**Comment by blowekamp** [comment]
@dave3d The test can be run via `ctest -V -R Python.Test.BasicFiltersTests`.

**Review by dave3d — APPROVED** [review]

File: Wrapping/Python/sitkImage.i [hunk 3]
```diff
@@ -566,11 +585,16 @@
...
             If all indices are integers then rvalue should be a pixel value
-            ( scalar or sequence for vector pixels). Value is assigned pixel.
-
-            If all indices are slice indices then, rvalue should be an image
-            of matching size, dimension and type as the image and region. This
-            image's pixel are then assigned the rvalue image with the paste filter.
+            ( scalar or sequence for vector pixels). The value is assigned to
+            the pixel.
+
+            If the indices are slices or integers then, the PasteImageFilter is
```
Comment (zivy) [inline Wrapping/Python/sitkImage.i:591]: "...PasteImageFilter is is used..."

File: Wrapping/Python/sitkImage.i [hunk 4]
```diff
@@ -566,11 +585,16 @@
...
+            is used to assign values to this image. The rvalue can be an image
+            or a scalar constant value. When rvalue is an image it must be of
+            the smame pixel type and equal or lesser dimension than self. The
```
Comment (zivy) [inline Wrapping/Python/sitkImage.i:594]: smame -> same

**Review by zivy — APPROVED** [review]
Some minor comments, but really looks good.

File: Wrapping/Python/sitkImage.i (hunk 4 above)
Comment (blowekamp) [reply Wrapping/Python/sitkImage.i]: done

File: Wrapping/Python/sitkImage.i (hunk 3 above)
Comment (blowekamp) [reply Wrapping/Python/sitkImage.i:591]: done

**Pull request merged**